docs/articles/documentation/continuous-integration/bitbucket-pipelines.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: docs
+title: Run Tests in Bitbucket Pipelines
+permalink: /documentation/continuous-integration/bitbucket-pipelines.html
+---
+# Run Tests in Bitbucket Pipelines CI
+
+You can automatically run TestCafe tests as a part of your build process on [Bitbucket Pipelines CI](https://bitbucket.org/product/features/pipelines).
+
+Suppose you have a Bitbucket project for which you need to automatically run tests in the cloud when the project is modified. To do this, go through the following steps:
+
+* [Step 1 - Install TestCafe and Create Tests](#step-1---install-testcafe-and-create-tests)
+* [Step 2 - Enable Bitbucket Pipelines for Your Project](#step-2---enable-bitbucket-pipelines-for-your-project)
+  * [Option 1 - Use TestCafe Docker Image](#option-1---use-testcafe-docker-image)
+  * [Option 2 - Install TestCafe on a Docker Image](#option-2---install-testcafe-on-a-docker-image)
+* [Step 3 - Trigger a Bitbucket Pipelines Build](#step-3---trigger-a-bitbucket-pipelines-ci-build)
+
+> TestCafe provides an [example](https://github.com/DevExpress/testcafe/tree/master/examples/running-tests-in-chrome-using-bitbucket-pipelines-ci/) that shows how to run tests in Chrome with Bitbucket Pipelines CI.
+
+## Step 1 - Install TestCafe and Create Tests
+
+Install TestCafe [locally](../using-testcafe/installing-testcafe.md#locally) in your project and [create tests](../getting-started/README.md#creating-a-test).
+
+## Step 2 - Enable Bitbucket Pipelines for Your Project
+
+To enable Bitbucket Pipelines for your project, create a `bitbucket-pipelines.yml` file in the project's root folder.
+
+Bitbucket Pipelines CI allows you to avoid manual environment configuration and use Docker containers set up to work with Chrome and Firefox. You can [use a TestCafe Docker image](#option-1---use-testcafe-docker-image) with pre-installed software.
+However, if you already have a Docker image prepared to deploy your web application and run tests, you can [install TestCafe on this image before testing](#option-2---install-testcafe-on-a-docker-image).
+
+For more information about how to get started with Pipelines, see the [Atlassian Bitbucket support page](https://confluence.atlassian.com/bitbucket/get-started-with-bitbucket-pipelines-792298921.html).
+
+### Option 1 - Use TestCafe Docker Image
+
+Specify the `testcafe/testcafe` image name in the `image` field of the [bitbucket-pipelines.yml file](https://confluence.atlassian.com/bitbucket/configure-bitbucket-pipelines-yml-792298910.html). Then, configure a pipeline that installs the project's dependencies and triggers a custom TestCafe launcher when you push commits and create pull requests.
```

Comment:
Specify the `testcafe/testcafe` image name in the `image` field of the [bitbucket-pipelines.yml file](https://confluence.atlassian.com/bitbucket/configure-bitbucket-pipelines-yml-792298910.html).